DigitalOffice/Services/UserService/Commands/UserLoginCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+
+namespace LT.DigitalOffice.UserService.Commands
+{
+    public class UserLoginCommand : IUserLoginCommand
+    {
+        /*private readonly IValidator<UserCredentialsRequest> validator;
```

Comment:
Why is it documented?

Reply:
There is logic intersect with task UserService_AddLogin, I commented this temporaril. I will past code here, when UserService_AddLogin will be ready for merge.